install_deps.sh

```diff
@@ -5,6 +5,10 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   brew install yarn --without-node
   brew install jq zip
 else
+  # Use the default C / C++ compilers,
+  # because some makefiles default to CC:=gcc:
+  export CC=/usr/bin/cc
```

Comment:
Can you set these two compiler variables in a conditional block for `"$BUILDARCH" == "arm64"` so we are sure it doesn't mess up the other builds? If it means copying some of the `apt-get` commands into that block that's fine. The other builds are just very particular so I don't want to risk muddying the waters.